cypress/integration/examples/mainTest.spec.js

```diff
@@ -0,0 +1,18 @@
+describe("main test", () => {
+  it("open main site and check console", () => {
+    cy.visit("http://localhost:8080")
+    
+  });
+});
+
+Cypress.on('window:before:load', (win) => {
+  cy.spy(win.console, 'error');
```

Comment:
Can't it be?
cy.spy(win.console.error)?

Reply:
I found something about spy in docs. 
https://docs.cypress.io/api/commands/spy.html#Syntax
and correct usage:
`cy.spy(object, method)`